src/controllers/route.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const path = require('path');
+const router = express.Router();
+
+const home = require('../views/home');
+
+router.get('/', (req, res) => {
+   res.render('home', { activePage: {home: true}});
+});
+
```

Comment:
We pushed it now with other changes which we worked on since then :nerd_face: 
It's #30
Thanks!